eliot/_message.py

```diff
@@ -102,9 +102,8 @@ def write(self, logger=None, action=None):
             action = _defaultAction
         contents = self._contents.copy()
         contents["timestamp"] = self._timestamp()
-        if action is not None:
```

Comment:
So... I guess the `if` was unnecessary, but this change is unrelated to actual fix.